client/wfm/src/result/StepResult.ts

```diff
@@ -0,0 +1,35 @@
+import { STATUS } from '../status';
+import { Step } from '../step/Step';
+
+/**
+ * Output of a single {@link Step} in the context of a {@link WorkOrder}
+ */
+export interface StepResult {
+  /**
+   * Form data output of the {@link Step}'s execution
+   */
+  submission?: {
```

Comment:
We may even create submission interface to help people to create Steps and denote what should be send as submissions - extra fields etc.